frontend/store/network.ts

```diff
@@ -114,20 +122,36 @@ export const actions = {
         await dispatch('surplus/setup', undefined, { root: true });
         await dispatch('debt/setup', undefined, { root: true });
     },
-    async setupNetworks({ commit }: ActionContext<State, State>, isDev?: boolean) {
+    async setupNetworks({ getters, commit, dispatch }: ActionContext<State, State>, oldRpcUrl?: string) {
+        if (!getters.getRpcUrl) {
+            return;
+        }
         commit('setIsChangingNetwork', true);
         if (networkChangeTimeoutId) {
             clearTimeout(networkChangeTimeoutId);
         }
-        const { networks, defaultChainId, defaultNetwork } = await setupRpcUrlAndGetNetworks(
-            process.env.RPC_URL,
```

Comment:
We should still fallback to the `process.env.RPC_URL` (cause we still want to keep https://unified-auctions.makerdao.com running, where we provide our RPC url) this PR should not affect default production behaviour of the website